.github/cdk.context.github.json

```diff
@@ -37,9 +37,6 @@
     }
   },
   "testImagery": {
-    "deploy": true,
-    "config": {
-      "BUILD_FROM_SOURCE": false
-    }
```

Comment:
is this our example json?  We may want to keep this here as a reference of how to set the flag.

Reply:
No, this `cdk.context.json` is used only during the Github Workflow actions. In addition to that, `testImagery` is a class that does not rely on Dockerfile. It is simply uploading images to the s3 bucket, hence we can't really build from source like model runner container.